test/config/utility.cc

```diff
@@ -113,6 +113,19 @@ name: envoy.squash
     nanos: 0
 )EOF";
 
+const std::string ConfigHelper::READONLY_RBAC_FILTER =
```

Comment:
nit: given this is only going to be used in an extension integration test, I would not add this to test/config/utility.cc. Just define in your extension integration test and use there.